### PR TITLE
OCPBUGS-6455: updating net deps

### PR DESCRIPTION
**Pull request by jaypoulz**

### CODE REVIEW

File: vendor/github.com/openshift/api/operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml [hunk 1]
```diff
@@ -51,7 +51,6 @@ spec:
                 - csi.sharedresource.openshift.io
                 - diskplugin.csi.alibabacloud.com
                 - vpc.block.csi.ibm.io
-                - powervs.csi.ibm.com
```
Comment (jaypoulz) [inline vendor/github.com/openshift/api/operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml:54]: @Madhan-SWE @mkumatag The vendor update for this API concerns me a little. Is it safe to remove these, or does the openshift API need updating?

Comment (dharaneeshvrd) [reply vendor/github.com/openshift/api/operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml:54]: All the values for the enum is still there, just that line nums got changed a a little . And `filestore.csi.storage.gke.io` got added newly.

**Comment by jaypoulz** [comment]
@Madhan-SWE can you give the /approve label to this one as well?

**Pull request merged**